test-coverage.txt

```diff
@@ -0,0 +1,54 @@
+> react-scripts test --env=jest-environment-jsdom-sixteen "--coverage" "--watchAll=false"
+
+ PASS  src/tests/Fortune.test.jsx
+ PASS  src/tests/Header.test.jsx
+ PASS  src/tests/App.test.jsx
+ PASS  src/tests/MainMenu.test.jsx
+ PASS  src/tests/VideoHome.test.jsx
+ PASS  src/tests/VideoItem.test.jsx
+ PASS  src/tests/UserMenu.test.jsx
+ PASS  src/tests/Search.test.jsx
+ PASS  src/tests/globals.test.js
+ PASS  src/tests/DarkMode.test.jsx
+--------------------------|----------|----------|----------|----------|-------------------|
+File                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
+--------------------------|----------|----------|----------|----------|-------------------|
+All files                 |      100 |      100 |      100 |      100 |                   |
```

Comment:
Awesome, your test coverage is really good. I see you are using snapshots, a downside of snapshots is that they are tightly coupled to an application's output, this makes that any changes, even to insignificant parts of the output, can cause snapshot tests to fail, this is just something to keep in mind when deciding to use snapshots in the tests